src/Service/Bulk/BulkCreateUsersService.php

```diff
@@ -206,6 +210,8 @@ private function createBulkUserAssignmentData(
             }
         }
 
+        $this->awsS3CsvWriter->writeCsv();
```

Comment:
i think it is better to call in the command class itself. 
it is not related to the creation of students